src/orb.yaml

```diff
@@ -0,0 +1,156 @@
+---
+version: 2.1
+
+definitions:
+  port-parameter: &port-parameter
+    description: port
+    type: integer
+  localhost-parameter: &localhost-parameter
+    description: host
+    type: string
+    default: localhost
```

Comment:
Are you sure that localhost works? I tried it with application tests and it only works with *cassandra* instead *localhost*.

Reply:
Yes. See Wait * commands in this [build](https://circleci.com/gh/Cobliteam/alexstrasza-core/211)

Reply:
ok